src/Assets/projects.json

```diff
@@ -0,0 +1,65 @@
+[
+      {
+        "title": "WEATHER APP",
+        "id": "1",
+        "overlay": "Weather App",
+        "image": "/assetsImg/weatherApp.png",
```

Comment:
Your images are really nice, how did you create them?

Reply:
Thank you! It's canva, the easiest ever. Start a free account and then search for "tablet" "iphone" etc and choose a template you like. Take a printscreen and just "drag and drop" the pictures in the templates.

Reply:
https://www.canva.com/